examples/face_classification.py

```diff
@@ -0,0 +1,161 @@
+            visual = self.vision.wait_for(isa='text')
+            category = self.vision.encode(visual)
+            if not category:
+                category = selected_category
+            self.memory.store(eh=face.eh, es=face.es, nl=face.nl, mh=face.mh, category=category)
+            if self.demo_blending:
+                blended_chunk = self.memory.get_blended_chunk(slots='eh', distances=self._get_distances(),
+                                                              es=face.es, nl=face.nl, mh=face.mh, category=category)
+                print(blended_chunk)
+
+
+class FaceClassificationSimulation:
+    TRAINING_STIMULI = {1: [i for i in range(10)], 2: [8, 12, 17, 19, 22, 23, 25, 27, 28, 32]}
+    CONDITIONS = [Condition('1EF', TRAINING_STIMULI[1], sim_weights=(.19, .12, .25, .45)),
+                  Condition('1HF7', TRAINING_STIMULI[1], exe_num=7, sim_weights=(.15, .15, .29, .41)),
+                  Condition('2EF', TRAINING_STIMULI[2], sim_weights=(.108, .463, 0, .429)),
+                  Condition('2HF19', TRAINING_STIMULI[2], exe_num=19, sim_weights=(.126, .587, 0, .288))]
+    HUMAN_CORRECT = {
+        '1EF': [.968, .665, .937, .975, .875, .134, .261, .171, .089, .045, .529, .538, .258, .541, .430, .283, .778,
+                .145, .281, .780, .291, .277, .601, .289, .272, .490, .261, .386, .269, .114, .796, .538, .981, .126],
+        '1HF7': [.956, .594, .931, .944, .775, .1, .1, .15, .069, .006, .4, .488, .275, .425, .238, .269, .675,
+                 .094, .325, .681, .125, .269, .45, .219, .275, .413, .275, .144, .294, .094, .806, .300, .931, .05],
+        '2EF': [.660, .253, .456, .375, .843, .156, .855, .113, .717, .681, .711, .763, .937, .338, .525, .394, .081,
+                .381, .774, .095, .944, .925, .094, .633, .394, .744, .675, .844, .838, .318, .27, .283, .369, .488],
+        '2HF19': [.748, .151, .43, .415, .698, .101, .881, .126, .516, .541, .881, .656, .956, .516, .66, .679, .031,
+                  .189, .963, .069, .938, .969, .044, .811, .675, .855, .731, .844, .950, .222, .126, .169, .409, .264]}
+
+    def run(self, n_trials=80, output=False, real_time=False, print_results=False, show_experiment=True, demo_blending=False):
+
+        for condition in self.CONDITIONS:
+            selected_categories = Data(len(FaceClassificationTask.FACES))
+
+            for _ in range(n_trials):
+                env = Environment(window=(500, 500) if show_experiment else None)
+                task = FaceClassificationTask(env, condition, corrects=selected_categories)
+                agent = FaceClassificationAgent(env, condition, demo_blending=demo_blending)
+                World(task, agent).run(2400, output=output, real_time=real_time)
+
+            prob_cat1 = Result(selected_categories.proportion(1), self.HUMAN_CORRECT[condition.name])
+            # TODO is proportion ^^ what you would do to find the prob of selecting category 1?? unsure what it expects...
```

Comment:
I would assume you want the number of selections that were `1` divided by the total number of selections

Reply:
is that not what i did?

Reply:
Yes you did, sorry, I hadn't seen that yet when I made the comment but I should have updated when I saw it. I will resolve this